Resources/ServerInfo/Guidebook/ServerRules/GeneralEtiquette/Rule2-3.xml

```diff
@@ -1,7 +1,7 @@
-﻿<Document>
+<Document>
   # Rule 2.3
   You are playing a high-roleplay server; speak and act like a human being. While playing in this server, you will be held to a higher standard of roleplay. With this being the case, you are expected to actively engage in roleplay when it comes up.
-  - Do not use text speech such as "omg," "idk," "wtf," or "lol."
+  - Do not use text speech such as "omg," "idk," "wtf," or "lol." Exception: You may use text speach inside NanoChat messages.
```

Comment:
I think the following sounds a little better, but it's a nitpick in general.
`- Do not use text speech such as "omg," "idk," "wtf," or "lol," while physically speaking as your character. Using text speech in non-verbal ways is fine, like writing on paper or in PDAs.`